pkgs/by-name/he/hexpatch/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  python3,
+  stdenv,
+  gcc,
+  llvmPackages,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hexpatch";
+  version = "1.9.3";
+
+  src = fetchFromGitHub {
+    owner = "Etto48";
+    repo = "HexPatch";
+    rev = "v${version}";
+    hash = "sha256-yyYxRJ+o+Z5z7PmjcFCsahRXZ9JHFmGmituzGTxY6ec=";
+  };
+
+  cargoHash = "sha256-bckZtHzgKQow7l2Cg6Xi1oNk13FH2xKe8cbZ/Xj9E04=";
+
+  nativeBuildInputs =
+    [
+      cmake
+      python3
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isLinux [ gcc ]
+    ++ lib.optionals (stdenv.hostPlatform.isWindows || stdenv.hostPlatform.isDarwin) [
+      llvmPackages.libclang
+    ];
```

These should not be needed, as they are already provided by Nixpkgs' `stdenv`. I tried building this package on `x86_64-linux` and `aarch64-darwin` without these and it worked just fine.

```suggestion
  nativeBuildInputs = [
    cmake
    python3
  ];
```